recipes/longshot/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "v0.3.2" %}
+# this file is taken from the rust-bio-tools recipe
+# https://github.com/bioconda/bioconda-recipes/blob/master/recipes/rust-bio-tools/meta.yaml
+
+package:
+  name: longshot
+  version: {{version}}
+
+build:
+  number: 0
+
+source:
+  url: https://github.com/pjedge/longshot/archive/{{ version }}.tar.gz
+  sha256: ec8ebfb6007d42c6952bac6843e903f899f33d6c0fadbd976cc272d8d7f4b279
+
+requirements:
+  build:
+    - rust >=1.32
+    - make
+    - {{ compiler('c') }}  # [not osx]
+    - {{ compiler('cxx') }}  # [not osx]
+    - clang_osx-64 # [osx]
+    - clangxx_osx-64 # [osx]
```

Comment:
Can't we use the compiler function here as well? I think the default compiler of macos is clang anyway.

Reply:
I removed those and left just the compiler function -- passed tests. thanks!